modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -66,26 +66,31 @@ ifdef::aws-sts[]
 $ oc adm release extract \
 --credentials-requests \
 --cloud=aws \
---to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+--to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+--quay.io/<path_to>/ocp-release:<version>
 ----
 endif::aws-sts[]
 ifdef::google-cloud-platform[]
 ----
 $ oc adm release extract \
 --credentials-requests \
 --cloud=gcp \
---to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+--to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+--quay.io/<path_to>/ocp-release:<version>
 ----
 endif::google-cloud-platform[]
 ifdef::alibabacloud-default,alibabacloud-customizations[]
 ----
 $ oc adm release extract \
 --credentials-requests \
 --cloud=alibabacloud \
---to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+--to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+--quay.io/<path_to>/ocp-release:<version>
 ----
 endif::alibabacloud-default,alibabacloud-customizations[]
 +
+<1> `/credrequests` is the location where the list of `CredentialsRequest` objects is stored. This command creates the path to the `credrequests` directory if it does not exist.
```

Comment:
Consider this replacement for the second sentence:

This command creates the `credrequests` directory if it does not exist.